server/lib/renderers/xlsx.js

```diff
@@ -93,6 +92,16 @@ function render(data) {
   return wb.writeToBuffer();
 }
 
+
+/*
+ find available data to write in the excel file
+ rows is the default key, if nothing found, this function get look at rowsDataKey
+ it return an empty array if no data found
+*/
+function find(data, _options) {
+  const options = _options || {};
+  return data.rows || data[options.rowsDataKey] || [];
```

Comment:
Should the check for `data[options.rowDataKey]` come first?  What should happen if we do this:
```js
const data = {
  rows : 'This is a cool example',
  students : [{ id : 1, name : 'josh' }, { id : 3, name : 'jose' }]
};

const values = find(data, { rowsDataKey: 'students' });
// values is 'This is a cool example'
```

Is this the expected result?

Reply:
Check the Reportmanager.js for this case, ```data.rows``` will be equal to ```data.students```
The man problem is, we have several final result, the soft option for me is the specify for all reports the rowsDataKey option